eclair-node/src/main/scala/fr/acinq/eclair/payment/LocalPaymentHandler.scala

```diff
@@ -43,5 +56,9 @@ class LocalPaymentHandler extends Actor with ActorLogging {
       sender ! CMD_FAIL_HTLC(htlc.id, Right(UnknownPaymentHash), commit = true)
 
   }
-
 }
+
+object LocalPaymentHandler {
+  def props(nodeParams: NodeParams) = Props(new LocalPaymentHandler(nodeParams))
+  case class NewPaymentRequest(amountMsat: MilliSatoshi)
```

Comment:
For more consistency:
-  `NewPaymentRequest` -> `ReceivePayment`
- `CreatePayment` -> `SendPayment`